content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -257,6 +257,20 @@ Your applications cannot expect any performance SLAs (disk IOPS for example)
 from local ephemeral storage.
 {{< /caution >}}
 
+{{< note >}}
+This is an exception for enforcing resource quota on ephemeral-storage.
+
+To make the resource quota work on ephemeral-storage, two things need to be done.
+
+* an admin sets the resource quota for ephemeral-storage
+* User needs to mention the ephemeral-storage resource limit in the Pod spec.
+
+If the user doesn't specify the ephemeral-storage resource limit in the pod spec,
+the resource quota is not enforced on ephemeral-storage. Thus the user can use
+any amount of ephemeral-storage.
+
```

Comment:
Am I right that:
this detail about ResourceQuota would apply to all resources, and not just to ephemeral storage?

If so, let's reframe it as a general detail rather than a specific one.

Reply:
@sftim when we enforce resourceQuota on ephemeral-storage it shows different behaviour and  to be precise it doesn't enforce resouce limits/request  and that is not mention  in documentation. This PR document what steps should we follow if we want to enforce resource quota on ephemeral-storage.